tests/tools/oecert/README.md

```diff
@@ -23,6 +23,10 @@ where Options are:
     --log LOG_FILENAME    : log file name, default: oecert.log
     --verbose             : dump verbose info of evidence
 
+For example:
+
+    host/oecert --evidence --verify --verbose
```

Comment:
Remove "host/". Both host and enclave files are in the same directory. Also, give a few more examples and simple description.

Reply:
Updated.